src/main/java/com/flowingcode/vaadin/addons/localecombobox/LocaleComboBox.java

```diff
@@ -42,18 +42,74 @@ public class LocaleComboBox extends ComboBox<Locale> {
   private static final String ITEM_LAYOUT_CLASS_NAME = "fc-locale-combo-box-item-layout";
   private static final String ITEM_FLAG_CLASS_NAME = "fc-locale-combo-box-item-flag";
 
+  /**
+   * Constant for the default display mode.
+   * <p>
+   * In this mode, the Locale names are displayed using the default locale's display format.
+   */
+  public static final int DISPLAY_DEFAULT = 0;
```

Comment:
Use an enum instead of magic constants.

Reply:
Changed the constants to a new enum inside the class and also updated the javadoc comments